includes/frontend.php

```diff
@@ -552,9 +544,17 @@ public function register_styles() {
 
 		if ( ! Plugin::$instance->experiments->is_feature_active( 'e_dom_optimization' ) ) {
 			// If The Dom Optimization feature is disabled, register the legacy CSS
+			$frontend_legacy_file_name = 'frontend-legacy' . $direction_suffix . $min_suffix . '.css';
+
+			if ( $has_custom_file ) {
+				$frontend_legacy_file_url = $this->get_custom_frontend_file_url( $frontend_legacy_file_name );
+			} else {
+				$frontend_legacy_file_url = ELEMENTOR_ASSETS_URL . 'css/' . $frontend_legacy_file_name;
+			}
```

Comment:
Maybe better to move it to the same function that generates the file URL and the logic will be part of that function?

Reply:
Done